tests/providers/elasticsearch/log/test_es_task_handler.py

```diff
@@ -38,6 +39,19 @@
 from .elasticmock import elasticmock
 
 
+def get_ti(dag_id, task_id, execution_date, create_task_instance):
+    ti = create_task_instance(
+        dag_id=dag_id,
+        task_id=task_id,
+        execution_date=execution_date,
+        dagrun_state=DagRunState.RUNNING,
+        state=TaskInstanceState.RUNNING,
+    )
+    ti.try_number = 1
+    ti.raw = False
+    return ti
```

Comment:
How about turning this into a fixture?

```python
@pytest.fixture()
def create_running_task_instance(create_task_instance):
    def _create_ti(**kwargs):
        ti = create_task_instance(
            dagrun_state=DagRunState.RUNNING,
            state=TaskInstanceState.RUNNING,
            **kwargs,
        )
        ti.try_number = 1
        ti.raw = False
        return ti

    return _create_ti
```

Reply:
it _was_ a fixture but i pulled it out so i could make a TI with diff params...

but i guess you can parametirize a fixture [like so](https://stackoverflow.com/a/33879151/2761682)?

Reply:
You can but it shouldn’t be generally be needed, it’s easier to make the fixture return a _function_ that takes arguments instead (like this one here). I searched your changes and this implementation seems to be good enough for the usages in this PR. You’d do

```python
@pytest.fixture()
def ti(self, create_running_task_instance):
    yield create_running_task_instance(
        dag_id=self.DAG_ID,
        task_id=self.TASK_ID,
        execution_date=self.EXECUTION_DATE,
    )
    clear_db_runs()
    clear_db_dags()
```

Reply:
i'm confused @uranusjr 
i'm not seeing how this fixture helps me.
there is already a  fixture here like this now.  i just pulled out a portion of it (and still use it in the existing fixture) but i just want to be able to specify a different execution date in my specific test than the one used by the fixture.

Reply:
if you are saying i should just change the fixture so that it returns a `create_ti(execution_date)` funciton and update  all the other tests call that function (insstead of just using a returned TI) then i can do -- lemme know